packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -41,36 +47,66 @@ export type RadioProps = {
    * Callback fired when the state is changed. Provides event and checked value.
    */
   onChange?: MuiRadioProps["onChange"];
+  /**
+   * Callback fired when the blur event happens. Provides event value.
+   */
+  onBlur?: MuiFormControlLabelProps["onBlur"];
 } & Pick<FieldComponentProps, "isDisabled" | "name"> &
-  SeleniumProps;
+  AllowedProps;
 
 const Radio = ({
+  inputFocusRef,
   isChecked,
   isDisabled,
   isInvalid,
   label,
   name,
   testId,
+  translate,
   value,
   onChange: onChangeProp,
+  onBlur: onBlurProp,
 }: RadioProps) => {
+  const ref = useRef<HTMLInputElement>(null);
+  useImperativeHandle(
+    inputFocusRef,
+    () => {
+      const element = ref.current;
+      return {
+        focus: () => {
+          element && element.focus();
```

Comment:
To match the rest of the code, can we change this to an explicit `if`?

```suggestion
          if (element) {
            element.focus();
          }
```

Reply:
Even better:
```suggestion
          element?.focus();
```